ajax/libs/shopify-buy/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "shopify-buy",
+  "description": "The JS Buy SDK is a lightweight library that allows you to build ecommerce into any website. It is based on Shopify's API and provides the ability to retrieve products and collections from your shop, add products to a cart, and checkout.",
+  "filename": "shopify_buy.min.js",
```

Comment:
Looks like its filename should be `shopify-buy.umd.min.js`?

Reply:
@PeterDaveHello Changed accordingly.